src/main/java/core/basesyntax/shop/InsufficientGoodsException.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.shop;
+
+public class InsufficientGoodsException extends Exception {
```

Comment:
Not sure you need your own exception.

Reply:
It's special dao-level exception, so maybe let it be